ketchup/options.py

```diff
@@ -16,17 +18,21 @@ def calculate_episodes_from_hours(episodes, episode_duration_mins=24):
 
 # Option 0 in Discord.
 def calc_watch_time_hrs(total_eps, episode_length_mins):
-    overall_watch_time = int(total_eps * episode_length_mins) / 60
+    overall_watch_time = total_eps * episode_length_mins / 60
```

Comment:
Removed the input being taken as an integer, because it was resulting in assertion errors later in the tests.